docs/dam/ibexa_dam.md

```diff
@@ -0,0 +1,83 @@
+---
+description: DAM module interface for uploading and managing image assets.
+edition: headless
+---
+
+# Ibexa DAM
+
+Digital Asset Management is a platform dedicated to editors. It enables storing in central location, organizing, distributing, 
+and sharing media assets across many channels.
+
+[[= product_name_dam =]] image picker is developed to work with any web browser that supports modern standards. However for the best user experience in editorial work
+it's recommended to use 1366x768 screen resolution.
```

Comment:
Are we sure about this recommended screen resolution? Isn't this the minimum resolution?

Reply:
It's taken from spec docs :)

Reply:
Not really. Spec describes minimum screen resolution. IMO, minimum is not the same as recommended.